# CustomMsg.msg
geometry_msgs/PoseStamped pose_stamped
geometry_msgs/Pose pose
nav_msgs/Odometry odometry